### PR TITLE
Non editable blocks showing

### PR DESCRIPTION
**Pull request by thiagomurakami**
#### What problem is this solving?

Blocks such as `Columns` that are supposed to be hidden are showing in the components list.

#### How should this be manually tested?

[Workspace](https://noneditableblocks--instore.myvtex.com/admin/cms/storefront)

#### Type of changes

| ✔️  | Type of Change                                                                            |
| --- | ----------------------------------------------------------------------------------------- |
| ✔️  | Bug fix                                |
| \_  | New feature                        |
| \_  | Breaking change  |
| \_  | Technical improvements  |

#### Notes

[ch15264]

### CODE REVIEW

File: react/components/EditorContainer/Sidebar/utils.test.ts [hunk 1]
```diff
@@ -140,6 +150,78 @@ describe('getComponents', () => {
...
+        getComponents(
+          {
+            'store/home': {
+              component: 'vtex.LayoutContainer',
+              props: {
+                elements: ['header', 'layout'],
+              },
+            },
+            'store/home/header': {
+              component: 'vtex.schema-without-title',
```
Comment (alcar) [inline react/components/EditorContainer/Sidebar/utils.test.ts]: ⛏ Perhaps `vtex.schema-with-title`?

File: react/components/EditorContainer/Sidebar/utils.test.ts [hunk 2]
```diff
@@ -140,6 +150,78 @@ describe('getComponents', () => {
...
```
Comment (alcar) [inline react/components/EditorContainer/Sidebar/utils.test.ts]: ⛏ Perhaps `vtex.schema-with-title`?

File: react/components/form/Dropdown.tsx [hunk 3]
```diff
@@ -8,10 +8,11 @@ import { CustomWidgetProps } from './typings'
 interface Props extends CustomWidgetProps, InjectedIntlProps {
   onClose?: () => void
   onOpen?: () => void
-  options: {
-    emptyValue: string
-    enumOptions: Array<{ label: string }>
-  }
+  // options: {
```
Comment (athoscouto) [inline react/components/form/Dropdown.tsx]: Is it supposed to stay as a comment?

File: react/components/form/Dropdown.tsx [hunk 4]
```diff
@@ -30,11 +31,24 @@ const Dropdown: React.FunctionComponent<Props> = ({
...
 
+  const dropdownOptions = React.useMemo(
+    () =>
+      Array.isArray(options.enumOptions)
+        ? options.enumOptions.map(option => ({
+            ...option,
+            label: formatIOMessage({ id: `${option.label}`, intl }),
+          }))
+        : [],
+    [options]
```
Comment (alcar) [inline react/components/form/Dropdown.tsx]: Wouldn't `[options.enumOptions]` suffice?

File: react/components/form/Dropdown.tsx (hunk 3 above)
Comment (thiagomurakami) [reply react/components/form/Dropdown.tsx]: Nope :shame:

File: react/components/EditorContainer/Sidebar/utils.test.ts (hunk 1 above)
Comment (thiagomurakami) [reply react/components/EditorContainer/Sidebar/utils.test.ts]: The component schema doesn't have a title... In fact, there isn't even a component, so I guess I could change that.

File: react/components/EditorContainer/Sidebar/utils.test.ts (hunk 2 above)
Comment (thiagomurakami) [reply react/components/EditorContainer/Sidebar/utils.test.ts]: The schema doesn't have a title:
```typescript
{
  'vtex.schema-without-title': { schema: {} },
} as any
```

**Pull request merged**